### PR TITLE
New cheat sheet: bootstrap3

### PR DESCRIPTION
**Pull request by raj-maurya**
## Type of Change

- [X] New Instant Answer
  - [X] Cheat Sheet
- [ ] Improvement
  - [ ] Bug fix
  - [ ] Enhancement
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.)
## Description of new Instant Answer, or changes

Bootstrap 3 quick and complete explanation of Classes like Helper Classes, Carousel, Alerts, Badges, Buttons and much more.

## People to notify

---

Forum Topic:

IA Page: https://duck.co/ia/view/bootstrap_3_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@raj-maurya Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/bootstrap3.json [hunk 1]
```diff
@@ -0,0 +1,594 @@
+{
+    "id": "bootstrap_3_cheat_sheet",
+    "name": "Bootstrap 3",
+    "description": "Bootstrap 3 quick explanation of Classes like Helper Classes, Carousel, Alerts, Badges, Buttons and much more.",
+    "metadata": {
+        "sourceName": "Bootstrap",
+        "sourceUrl": "http://getbootstrap.com/"
+    },
+    "aliases": [
+
```
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/bootstrap3.json]: Extra line

File: share/goodie/cheat_sheets/json/bootstrap3.json [hunk 2]
```diff
@@ -0,0 +1,594 @@
...
+        "twitter bootstrap",
+        "bootstrap3",
+        "bootstrap version 3"
+    ],
+    "template_type": "code",
+    "section_order": [
+
```
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/bootstrap3.json]: Extra line here as well

File: share/goodie/cheat_sheets/json/bootstrap3.json [hunk 3]
```diff
@@ -0,0 +1,594 @@
+{
+    "id": "bootstrap_3_cheat_sheet",
+    "name": "Bootstrap 3",
+    "description": "Bootstrap 3 quick explanation of Classes like Helper Classes, Carousel, Alerts, Badges, Buttons and much more.",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/bootstrap3.json]: Lets shorten the description

**Comment by pjhampton** [comment]
Looks like this PR has gone stale. Closing for now

**Pull request closed without merging**